molviewspec/molviewspec/builder.py

```diff
@@ -725,6 +763,75 @@ def transparency(
         return self
 
 
+class VolumeRepresentation(_Base):
+    """
+    Builder step with operations relating to particular representations.
+    """
+
+    # def color_from_uri(
+    #     self,
+    #     *,
+    #     schema: SchemaT,
+    #     uri: str,
+    #     format: str,
+    #     category_name: str | None = None,
+    #     field_name: str | None = None,
+    #     block_header: str | None = None,
+    #     block_index: int | None = None,
+    #     additional_properties: AdditionalProperties = None,
+    # ) -> Representation:
+    #     """
+    #     Use another resource to define colors of this representation.
+    #     :param schema: granularity/type of the selection
+    #     :param uri: resource location
+    #     :param format: format ('cif', 'bcif', 'json') of the content
+    #     :param category_name: only applies when format is 'cif' or 'bcif'
+    #     :param field_name: name of the column in CIF or field name (key) in JSON that contains the desired value (color/label/tooltip/component...); the default value is 'color'/'label'/'tooltip'/'component' depending on the node kind
+    #     :param block_header: only applies when format is 'cif' or 'bcif'
+    #     :param block_index: only applies when format is 'cif' or 'bcif'
+    #     :param additional_properties: optional, custom data to attach to this node
+    #     :return: this builder
+    #     """
+    #     params = make_params(ColorFromUriParams, locals())
+    #     node = Node(kind="color_from_uri", params=params, additional_properties=additional_properties)
+    #     self._add_child(node)
+    #     return self
+
+    def color(
+        self,
+        *,
+        color: ColorT,
+        selector: ComponentSelectorT | ComponentExpression | list[ComponentExpression] = "all",
+        additional_properties: AdditionalProperties = None,
+    ) -> Representation:
+        """
+        Customize the color of this representation.
+        :param color: color using SVG color names or RGB hex code
+        :param selector: optional selector, defaults to applying the color to the whole representation
+        :param additional_properties: optional, custom data to attach to this node
+        :return: this builder
+        """
+        params = make_params(ColorInlineParams, locals())
+        node = Node(kind="color", params=params, additional_properties=additional_properties)
+        self._add_child(node)
+        return self
```

Comment:
I don't think `selector` makes sense here, as we have no way of applying our current selectors to volumes.
At least for now. 
In the future we might add different types of selectors, which would apply to volumes.